resources/js/components/PMColumnFilterPopover/PMColumnFilterForm.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="pm-filter-form">
+    <b-form>
+      <b-form-group>
+        <b-button variant="light"
```

Comment:
maybe onSortAscending and onSortDesending buttons could abstract as a dynamic component with custom params  (svg icon, title, click handler and toggle event)